src/LibChorus/merge/xml/generic/Conflict.cs

```diff
@@ -152,7 +154,7 @@ public void WriteAsChorusNotesAnnotation(XmlWriter writer)
 			writer.WriteAttributeString(@"author", string.Empty, Author);
 			writer.WriteAttributeString(@"status", string.Empty, @"open");
 			writer.WriteAttributeString(@"guid", string.Empty, Guid.ToString());//nb: ok to have the same guid with the conflict, as they are in 1-1 relation and eventually we'll remove the one on conflict
-			writer.WriteAttributeString(@"date", string.Empty, DateTime.UtcNow.ToString(TimeFormatNoTimeZone));
+			writer.WriteAttributeString(@"date", string.Empty, DateTime.UtcNow.ToString(TimeFormatWithTimeZone));
```

Comment:
Is the intent to go to all local time? This file still uses DateTime.UtcNow throughout.